### PR TITLE
Update the TestCafe configuration, add servicePort1 and servicePort2 instead of servicePort (fix #9)

### LINKED ISSUE #9: Update the TestCafe configuration, add servicePort1 and servicePort2 instead of servicePort
http://testcafe.devexpress.com/Documentation/API_Reference/Continuous_Integration_API/TestCafe_Constructor/

### PR DESCRIPTION
**Pull request by LavrovArtem**
\сс @churkin @VasilyStrelyaev

### CODE REVIEW

**Comment by churkin** [comment]
lgtm

File: README.md [hunk 1]
```diff
@@ -19,7 +19,8 @@ Module initialization
 	* `TestCafe`: TestCafe Configuration
 		* `hostname`: The hostname of your computer
 		* `controlPanelPort`: The port number where you can access TestCafe Control Panel
-		* `servicePort`: The port number used by TestCafe to perform testing
+		* `servicePort1`: The first port number used by TestCafe to perform testing
+		* `servicePort2`: The second port number used by TestCafe to perform testing
```
Comment (VasilyStrelyaev) [inline README.md]:
```
		* `servicePort1`: The number of the first port used to perform testing
		* `servicePort2`: The number of the second port used to perform testing
```

Comment (VasilyStrelyaev) [inline README.md]: Or is there any significant difference between them?

Comment (LavrovArtem) [inline README.md]: nope

**Comment by LavrovArtem** [comment]
fpr

**Comment by VasilyStrelyaev** [comment]
lgtm

**Pull request merged**